contracts/interchain-token/src/contract.rs

```diff
@@ -0,0 +1,41 @@
+use axelar_soroban_std::ensure;
+use soroban_sdk::{contract, contractimpl, Address, Env};
+
+use crate::error::ContractError;
+use crate::event;
+use crate::storage_types::DataKey;
+
+#[contract]
+pub struct InterchainToken;
+
+#[contractimpl]
+impl InterchainToken {
+    pub fn initialize(env: Env, owner: Address) -> Result<(), ContractError> {
+        ensure!(
+            env.storage()
+                .instance()
+                .get::<DataKey, bool>(&DataKey::Initialized)
+                .is_none(),
+            ContractError::AlreadyInitialized
+        );
+
+        env.storage().instance().set(&DataKey::Initialized, &true);
+
+        env.storage().instance().set(&DataKey::Owner, &owner);
+
+        Ok(())
+    }
+
+    pub fn owner(env: &Env) -> Address {
+        env.storage().instance().get(&DataKey::Owner).unwrap()
+    }
```

Comment:
since Owner is set in `initialize`, this should be fine to avoid a panic
```suggestion
    pub fn owner(env: &Env) -> Result<Address, ContractError> {
        env.storage().instance().get(&DataKey::Owner).ok_or(ContractError::NotInitialized)
    }
```
i've followed this pattern in other locations